loopd/run.go

```diff
@@ -237,3 +227,25 @@ func Run(rpcCfg RPCConfig) error {
 
 	return fmt.Errorf("unimplemented command %v", parser.Active.Name)
 }
+
+// getConfigPath gets our config path based on the values that are set in our
+// config.
+func getConfigPath(cfg Config, loopDir string) string {
+	// If the config file path provided by the user is set, then we just
+	// use this value.
+	if cfg.ConfigFile != defaultConfigFile {
+		return lncfg.CleanAndExpandPath(cfg.ConfigFile)
+	}
+
+	// If the user has set a loop directory that is different to the default
+	// we will use this loop directory as the location of our config file.
+	// We do not namespace by network, because this is a custom loop dir.
+	if loopDir != loopDirBase {
```

Comment:
Thinking a bit about this, I think we could still namespace by network to make this even more simple.

Reply:
Also thought about this, but if you're setting your own base dir, I think it makes sense that we leave everything up to the user? I can imagine using base dir to tell loop that everything it needs is in one place, so I wouldn't want to then have to include network in my own setup.